.github/workflows/pull-request-test.yml

```diff
@@ -69,11 +78,13 @@ jobs:
       env:
         KUBECONFIG: ${{ github.workspace }}/renkubot-kube.config
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
-        CI_RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+        CI_RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES_ANON }}
```

Comment:
do we need this for every CI? I would like to avoid having even more namespaces and more resources used when they are not absolutely needed.
Would it make sense to use a label for the PRs that want to include this feature and chose the appropriate CI_RENKU_VALUES and not do it by default?